recipes-core/initscripts/initscripts-1.0/functions-dbus

```diff
@@ -0,0 +1,29 @@
+# -*-Shell-script-*-
+#
+# functions-dbus
+# 	This file contains DBus functions to be used by most or all
+# 	shell scripts in the /etc/init.d directory.
+#
+
+# Usage: dbus_name_has_owner NAME
+# Returns an error code if the service has not been claimed, else success.
+dbus_name_has_owner () {
+    local name="$1"
+
+    dbus-send --system --type=method_call --print-reply \
+        --dest=org.freedesktop.DBus /org/freedesktop/DBus \
+        org.freedesktop.DBus.NameHasOwner \
+        string:"com.citrix.xenclient.db" >/dev/null 2>&1
```

Comment:
`com.citrix.xenclient.db` should be `$name`?

Reply:
My apologies, I messed up something, I thought I had attended that already.
I will refresh the branch today.